project/api/views.py

```diff
@@ -118,6 +119,65 @@ class SpacetimeVolumeViewSet(viewsets.ModelViewSet):
     )
     serializer_class = SpacetimeVolumeSerializer
 
+    @transaction.atomic
+    def create(self, request, *args, **kwargs):
+        """
+        Solve overlaps if included in request body
+        """
+
+        # TODO: need to check that `territory` actually overlaps
+        if "overlaps" in request.data:
+            geom = GEOSGeometry(str(request.data["territory"]))
+            overlaps_db = SpacetimeVolume.objects.raw(
+                """
+                SELECT id, territory FROM api_spacetimevolume as stv WHERE (ST_IsEmpty((
+                    SELECT geom FROM (
+                        SELECT
+                            (ST_Dump(ST_Intersection(orig, diff))).geom as geom
+                        FROM (
+                            SELECT *
+                                , poly1 as orig, ST_Difference(poly2, poly1) as diff
+                            FROM (
+                                SELECT
+                                    stv.territory as poly1
+                                    , ('{}'::geometry) as poly2
+                            ) as foo
```

Comment:
I would suggest to add filter by dates that should reduce the number of polygons to compare with. 
```suggestion
                                WHERE stv.end_date >= {start_date}::numeric(10,1) AND stv.start_date <= {end_date}::numeric(10,1)
                            ) as foo
```